components/kserve/kserve.go

```diff
@@ -229,3 +233,53 @@ func checkDepedentOps(cli client.Client) *multierror.Error {
 	}
 	return multiErr
 }
+
+func (k *Kserve) configureServiceMesh(cli client.Client, dscispec *dsciv1.DSCInitializationSpec, enabled bool) error {
```

Comment:
I think both suggestions are mutually exclusive, since by moving the code to `servicemesh_setup.go` would require the `enabled` argument (since it is two different `packages`).

Moving the code can be debatable, since mesh setup for KServe, while the other file is mesh setup barebones?

For now, I just applied the first suggestion. Not sure about the 2nd one.

Reply:
@israel-hdez what I meant is to move this part to its own file in this package instead of keeping it in in `kserve.go`.

Reply:
Ah, that makes sense! I followed your suggestion.